test/mailers/previews/user_mailer_preview.rb

```diff
@@ -40,6 +40,18 @@ def claim_notification_registered
     UserMailer.claim_notification(creator_id, [work.id], true)
   end
 
+  def admin_hidden_bookmark_notification
+    bookmark = create(:bookmark)
+    creator_id = bookmark.pseud.user_id
+    UserMailer.admin_hidden_bookmark_notification(bookmark.id, creator_id)
```

Comment:
It'd be nice to have a more "standard" username in the preview:
```suggestion
    creator = create(:user, :for_mailer_preview)
    bookmark = create(:bookmark, pseud: creator.default_pseud)
    UserMailer.admin_hidden_bookmark_notification(bookmark.id, creator.id)
```